core/i18n/resource/i18n/pt-PT.csv

```diff
@@ -0,0 +1,43 @@
+"No such configuration for the product. Please do choose another combination of attributes.","Esta configuração não é possível para este produto. Por favor selecione outra configuração de opções."
+"The system is not sure about the stock quantity (volatile). Product has been added to the cart for pre-reservation.","O sistema não tem certeza sobre a quantidade em stock (volátil). O produto foi adicionado ao Cesto de Compras em pré-reserva."
+"This feature is not implemented yet! Please take a look at https://github.com/DivanteLtd/vue-storefront/issues for our Roadmap!","Esta funcionalidade ainda não foi implementada! Por favor consulte o nosso Rodmap em https://github.com/DivanteLtd/vue-storefront/issues!"
+"The product is out of stock and cannot be added to the cart!","O produto não se encontra em stock e não pode ser adicionado ao Cesto de Compras!"
+"Product has been added to the cart!","O produto foi adicionado ao Cesto de Compras!"
+"Internal validation error. Please check if all required fields are filled in. Please contact us on contributors@vuestorefront.io","Erro interno de validação. Por favor verifique se todos os campos obrigatórios estão preenchidos. Por favor contate-nos em contributors@vuestorefront.io"
+"Product {productName} has been added to the compare!","Produto {productName} foi adicionado a Comparar Produtos!"
+"Product {productName} has been removed from compare!","Produto {productName} foi removido de Comparar Produtos!"
+"Product {productName} has been added to wishlist!","Produto {productName} foi adicionado à Lista de Desejos!"
+"Product {productName} has been removed from wishlit!","Produto {productName} foi removido da Lista de Desejos!"
+"Account data has successfully been updated","Os dados da conta foram atualizados"
+"Newsletter preferences have successfully been updated","As suas preferências para a Newsletter foram atualizadas."
+"Reset password feature does not work while offline!","A funcionalidade de Recuperação de Senha não funciona sem ligação à internet!"
+"You are logged in!","Você está com sessão inicializada!"
+"Please fix the validation errors","Por favor corrija os erros de validação"
+"Product price is unknown, product cannot be added to the cart!","O preço do produto é desconhecido, o produto não pode ser adicionado ao Cesto de Compras!"
+"My Account","A Minha Conta"
+"Type what you are looking for...","Escreva o que está procurando..."
+"Home Page","Página Inicial"
+"Checkout","Finalizar Compra"
+"Subtotal incl. tax","Subtotal c/ imposto"
+"Grand total","Total"
+"Field is required","Campo obrigatório"
+"Field is required.","Campo obrigatório."
+"You're logged out","Você não está com sessão inicializada"
```

Comment:
Talvez `Você está desconectado`

Reply:
Em certos casos "Está desconectado" também se usa, mas normalmente é ambíguo! Daí que o mais normal é usar "Está com sessão inicializada" para "You are logged in" e "Não está com sessão inicializada" para "You are logged out" (por vezes estas frases começam por 'Você', mas não é frequente...).